Sim/SimPapas/src/PapasSimulatorTool.h

```diff
@@ -17,7 +17,7 @@ class MCParticleCollection;
 /** @class PapasSimulatorTool Sim/SimPapas/src/PapasSimulatorTool.h PapasSimulatorTool.h
  *
  *  Runs the papas simulation algorithm. This creates a list of particles from Pythia generated particles.
```

Comment:
which particles are we talking about? 
"list of particles" not explicit enough

Reply:
done